manager/server/src/main/java/org/openremote/manager/server/security/ManagerIdentityService.java

```diff
@@ -231,6 +232,13 @@ public UserConfiguration mergeUserConfiguration(UserConfiguration userConfigurat
         return persistenceService.doReturningTransaction(em -> em.merge(userConfiguration));
     }
 
+    public boolean isUserInTenant(String userId, String tenantId) {
```

Comment:
The parameter name should be realmId instead of tenantId, we use 

- "tenantRealm" or "realm" for the unique realm name
- "realmId" or "id" for Keycloak entity identifier
- "tenantDisplayName" or "displayName" for what is shown to users